metricbeat/module/iis/application_pool/_meta/data.json

```diff
@@ -0,0 +1,26 @@
+{
+    "@timestamp" : "2020-02-13T14:55:27.000Z",
+    "metricset" : {
+        "name" : "application_pool",
+        "period" : 10000
+    },
+    "service" : {
+        "type" : "iis"
+    },
+    "iis" : {
+        "application_pool" : {
+            "net_clr" : {
+                "total_exceptions_thrown" : 0
+            },
+            "name" : "test.local",
+            "process" : {
+                "pid" : 11956,
```

Comment:
I meant to use the ECS `process.pid`, at the root level. But if it complicates things with the permon metricset we can leave moving this field to ECS for the future.

Reply:
@jsoriano , indeed, my mistake, process.pid is a root filed now, application pool does not go through perfmon so that simplified the case.